docs/src/index.js

```diff
@@ -9,6 +9,8 @@ import ReactPerf from 'react-addons-perf';
 import { render } from 'react-dom';
 import App from 'docs/src/components/App';
 
+require('core-js');
```

Comment:
I think that we can address that on another PR. I would rather not use `Array.from` in that case.